packages/firebase_auth/test/firebase_auth_e2e.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:e2e/e2e.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Can sign-in anonymously', (WidgetTester tester) async {
+    final FirebaseAuth authUnderTest = FirebaseAuth.instance;
+    final AuthResult result = await authUnderTest.signInAnonymously();
```

Comment:
This plugin already has an integration test suite in examples/test/firebase_auth.dart. I'd like to migrate that here, instead of maintaining two separate test suites.

The test driver file in examples/test_driver/ should be updated to use the result code provided by the e2e plugin.